ql/patterns/observable.cpp

```diff
@@ -151,7 +151,8 @@ namespace QuantLib {
             }
         }
 
-        sig_->disconnect(ext::bind(&Observer::Proxy::update,
+        // signals2 needs boost::bind, std::bind does not work
+        sig_->disconnect(boost::bind(&Observer::Proxy::update,
```

Comment:
This might need including the corresponding Boost header if the rest of the library uses `std::bind`.